src/transformers/modeling_tf_bert.py

```diff
@@ -237,6 +237,16 @@ def _linear(self, inputs):
         return tf.reshape(logits, [batch_size, length, self.vocab_size])
 
 
+def DenseEinsum(output_size, kernel_initializer=None, name=None):
+    return tf.keras.layers.experimental.EinsumDense(
```

Comment:
Maybe check to see the TF version here?  We should support tf 2.0+ for the lib's internals